deepmd/fit/ener.py

```diff
@@ -470,7 +474,10 @@ def build(
         if input_dict is None:
             input_dict = {}
         bias_atom_e = self.bias_atom_e
-        type_embedding = input_dict.get("type_embedding", None)
+        if self.compress and isinstance(self.descrpt, deepmd.descriptor.DescrptSeAtten):
+            type_embedding = input_dict.get("type_embedding_from_graph", None)
```

Comment:
I suggest that `TypeEmbedNet.build` directly returns `type_embedding_from_graph`, and we don't need to hack `Fitting`